build.gradle

```diff
@@ -58,10 +67,23 @@ subprojects {
 
 		lambda "com.amazonaws:aws-lambda-java-core:${dependencyVersions.aws_lambda_core_version}",
 				"com.amazonaws:aws-lambda-java-events:${dependencyVersions.aws_lambda_events_version}",
-				"com.amazonaws:aws-lambda-java-log4j2:1.5.1",
 				"software.amazon.lambda:powertools-parameters:${dependencyVersions.aws_powertools_version}",
 				"org.apache.httpcomponents:httpclient:4.5.13"
 
+		lambda_tests "com.amazonaws:aws-lambda-java-tests:1.1.1"
+
+		log4j "org.apache.logging.log4j:log4j-api:2.17.2",
```

Comment:
we already have powertools on the main branch which gives us log4j. Do we need it again?